### PR TITLE
add computed isEnabled getter to Workspace, use it everywhere

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

alternative to #1965

### CODE REVIEW

**Comment by Nokel81** [comment]
closes #1965

File: src/common/workspace-store.ts [hunk 1]
```diff
@@ -83,6 +83,13 @@ export class Workspace implements WorkspaceModel, WorkspaceState {
     }
   }
 
+  /**
+   * Is this workspace not manged by an extension or is it managed and explicitly enabled
+   */
+  get isEnabled(): boolean {
```
Comment (jakolehm) [inline src/common/workspace-store.ts:89]: Why different name? Should we use `get enabled` and `set enabled` so that api stays the same?

Comment (Nokel81) [reply src/common/workspace-store.ts:89]: I left the `@observable enabled: boolean` as the current method of setting this. Having an `is***` getter seemed to me to be more in line with how our other getters are named. But I can change it if you want.

**Comment by jakolehm** [comment]
Duplicate of #1965 .

**Pull request closed without merging**